release-notes.md

```diff
@@ -1,3 +1,18 @@
+# Naos 14.11.0 #
+
+## Beacon indexes rotation ##
+- Issue Type: Task
+- Issue ID: PLAT-9288
+
+We added a script (plugins/beacon/scripts/BeaconsIndexesRotation.php) to support index rotation in our beacon service.
```

Comment:
Added a script ()........
Run this script as a monthly cron job if you want to delete your old beacon events.